src/modules/dashboard/ListDataCard.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardHeader,
+  Grid,
+} from '@mui/material'
+
+import { OptionalLink } from '@/components/links/optional-link'
+
+import { StyledAvatar } from './StyledAvatar'
+
+interface IListDataCardProps {
+  title: string
+  items: JSX.Element | JSX.Element[] | string
```

Comment:
```suggestion
  items: React.ReactNode
```

This is basically the same type you've written here by hand